docs/pages/guides/webhooks.mdx

```diff
@@ -0,0 +1,196 @@
+---
+meta:
+  title: "Webhooks"
+  parentTitle: "Guides"
+  description: "Learn Liveblocks Webhooks concepts and implementation"
+---
+
+## Webhooks
+
+In this guide, we will explain how to use webhooks within your product so that
+you can react to Liveblocks events as they happen. Now that our platform is
+observable, you can use webhooks to reduce development time and the need for
+polling.
+
+### What is a webhook?
+
+Webhooks make a platform observable. They let users subscribe to events on the
+platform for them to react to. In this sense, webhooks streamline the collection
+of data between two connected services.
+
+Like an API, webhooks are a mechanism for a client to receive data. The main
+difference between webhooks and APIs is that a webhook functions as an inverted
+API. A traditional API receives requests and sends responses, whereas a webhook
+is an endpoint that you provide where we send events.
+
+The lifecycle of a webhook:
+
+1. You create and provide a client endpoint, which listens to event messages
+2. The Liveblocks server broadcasts a message (event) to the client endpoint
+   when an event occurs
+3. You verify the event message payload from the webhook service provider
+   (Svix), and send a 200 response once the webhook has been verified
+4. You use the message payload to trigger a task
+
+Steps 2-4 will be repeated as long as you are subscribed to Liveblocks events.
+
+In this way, webhooks provide a real-time connection between two services. While
+APIs are useful for obtaining data on a scheduled basis, they are not efficient
+for observing changes on a platform that has a high volume of activity.
+
+### Concepts
+
+## Messages
+
+For each activity that occurs under the defined list of events within the Event
+Catalog a message is generated. Messages contain the metadata for each event
+that you may choose to subscribe to. They also contain event data and an
+alphanumeric identifier.
+
+## Events
+
+Messages sent through Svix each have an `eventType` property which is used to
+identify and filter what is being sent to the client. Each endpoint you provide
+in the webhooks dashboard listens to all events by default but can be easily
+configured to only listen to a subset by updating the Message Filtering section.
+As a new user, you may opt to listen to all events or subscribe to only the ones
+that are relevant to your use case.
+
+Alternatively, you can use the Event Catalog provided as a way to determine
```

Comment:
I feel we could have more screenshots, like here showing the Event Catalog. What do you think @taylorpwhipp ?

Reply:
I have the event catalog showed in the editing events section.  It also has more events that need to be added.  Let me know if you want a separate screenshot of it in current state